.github/workflows/start-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Trigger Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-setup:
+    name: Setup Release Branch
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+      id: checkout
+      with:
+        submodules: "recursive"
+        ref: "develop"
+    - name: Pull Tags
+      id: pull-tags
+      # We don't want a full heavyweight checkout with full history
+      # checkout action only can get tags + full history, so do this separately.
+      # Don't need tags in submodules.
+      run: git fetch --no-recurse-submodules origin 'refs/tags/*:refs/tags/*'
+    - name: Get latest version
+      id: latest-version
+      run: |
+        LATEST_TAG=$(git tag | sort -V | tail -n 1 | cut -c 2- )
+        echo ::set-output name=latest-version::${LATEST_TAG}
+    - name: Bump release version
+      id: next-version
+      uses: christian-draeger/increment-semantic-version@1.0.2
+      with:
+        current-version: "${{ steps.latest-version.outputs.latest-version }}"
+        version-fragment: 'feature'
```

Comment:
can we make this an input field? we do make patch releases

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs

```yaml
    inputs:
      bumpVersionFragment:
        description: 'Version fragment to bump'     
        required: true
        default: 'feature' 
        type: choice
        options:
        - feature
        - bug
```

I'd also exclude the major version to prevent an accidental 1.0 release before it's time :D